testing/logging/test_reporting.py

```diff
@@ -293,6 +294,62 @@ def test_log_2(fix):
     ])
 
 
+def test_sections_single_new_line_after_test_outcome(testdir, request):
+    """Check that with live logging enable we are printing the correct headers during
```

Comment:
It seems this docstring is not related to the test

Reply:
yeah, copy/paste